### PR TITLE
#11642 - Changes to Manifest file & other small updates.

### PR DESCRIPTION
**Pull request by pavanmanuguri**
# 11642

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

**Review by carmenberdant1 — CHANGES_REQUESTED** [review]
Hi @pavanmanuguri, please make sure all your file names are lowercase, and check your images to have the compartment blurred. Additionally, you need to have some URLs replaced, I will DM you about this.

File: ce-genai-ocw24/workshops/desktop/index.html
Comment (klazarz) [inline ce-genai-ocw24/workshops/desktop/index.html:1]: <a href="https://oracle.com/livelabs" target="_blank" id="livelabs" title="Oracle LiveLabs"></a>


the url livelabs.oracle.com

File: ce-genai-ocw24/workshops/sandbox/index.html
Comment (klazarz) [inline ce-genai-ocw24/workshops/sandbox/index.html:1]: <a href="https://oracle.com/livelabs" target="_blank" id="livelabs" title="Oracle LiveLabs"></a>


the url livelabs.oracle.com

File: ce-genai-ocw24/workshops/tenancy/index.html
Comment (klazarz) [inline ce-genai-ocw24/workshops/tenancy/index.html:1]: <a href="https://oracle.com/livelabs" target="_blank" id="livelabs" title="Oracle LiveLabs"></a>


the url livelabs.oracle.com

**Review by klazarz — CHANGES_REQUESTED** [review]
please make conflicts are resolved and all filenames are lower case

File: ce-genai-ocw24/workshops/sandbox/index.html
Comment (pavanmanuguri) [reply ce-genai-ocw24/workshops/sandbox/index.html:1]: updated

File: ce-genai-ocw24/workshops/tenancy/index.html
Comment (pavanmanuguri) [reply ce-genai-ocw24/workshops/tenancy/index.html:1]: updated

**Comment by klazarz** [comment]
@pavanmanuguri filenames are still not all lowercase. you need to either use new names or use 'git mv' to rename.

**Review by klazarz — COMMENTED** [review]
@pavanmanuguri please make sure index.html are correct:

   <a href="https://oracle.com/livelabs" target="_blank" id="livelabs" title="Oracle LiveLabs"></a>

The URL should be livelabs.oracle.com

**Review by klazarz — DISMISSED** [review]

**Comment by pavanmanuguri** [comment]
Resolved all conflicts, please complete this merge.

**Review by klazarz — APPROVED** [review]

**Pull request merged**